packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -331,6 +339,15 @@ export interface ICompressionRuntimeOptions {
  * @alpha
  */
 export interface IContainerRuntimeOptions {
+	/**
+	 * Compatibility mode is used to determine the default configuration for version-dependent options. It can be considered as the
```

Comment:
If I were an FF customer reading this, I would want a definition for "version-dependent option". (side note, not totally sure I like that name but it's somewhat reasonable). Not sure if that should be inline here or a reference to another type / explanation.